src/components/Index/Environment.vue

```diff
@@ -44,13 +57,20 @@
       <small class="form-text text-muted">Set your polling interval.</small>
     </div>
 
-    <div class="form-check mb-3" v-if="isEnabled(AllFeatures.bucketing)">
-      <input class="form-check-input" type="checkbox" v-model="bucketing" />
-      <label class="form-check-label"> Use bucketing </label>
+    <div v-if="flagshipMode == 'bucketing'" class="form-group">
```

Comment:
Here too, maybe another feature (to create in the global const of features) like pollingInternalUnit

Reply:
Yes, I agree

Reply:
I named it pollingIntervalUnit